### PR TITLE
Add REST/CLI interfaces to government

### PR DESCRIPTION
**Pull request by gsora**
This PR adds a CLI and a REST interface to easily retrieve the government address.

## Checklist
- [x] Targeted PR against correct branch
- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [x] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer

______

For Admin Use:
- [ ] Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- [ ] Reviewers Assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]"

### CODE REVIEW

File: x/government/client/cli/query.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	}
+}
+
+func getCmdGetGovernmentAddrFunc(cmd *cobra.Command, args []string, cdc *codec.Codec) error {
+	cliCtx := context.NewCLIContext().WithCodec(cdc)
+
+	route := fmt.Sprintf("custom/%s/%s", types.QuerierRoute, types.QueryGovernmentAddress)
+	res, _, err := cliCtx.QueryWithData(route, nil)
+	if err != nil {
+		fmt.Printf("Could not get government address: \n %s", err)
```
Comment (alessio) [inline x/government/client/cli/query.go]:
```suggestion
		cmd.Printf("Could not get government address: \n %s", err)
```

Comment (alessio) [reply x/government/client/cli/query.go]: Actually, this should return the error rathen than continue

Comment (alessio) [reply x/government/client/cli/query.go]:
```suggestion
		return fmt.Errorf("could not get government address: %s", err)
```

Comment (gsora) [reply x/government/client/cli/query.go]: Committed.

File: x/government/client/cli/query.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ func getCmdGetGovernmentAddrFunc(cmd *cobra.Command, args []string, cdc *codec.C
 	route := fmt.Sprintf("custom/%s/%s", types.QuerierRoute, types.QueryGovernmentAddress)
 	res, _, err := cliCtx.QueryWithData(route, nil)
 	if err != nil {
-		fmt.Printf("Could not get government address: \n %s", err)
+		return fmt.Errorf("could not get government address: \n %s", err)
```
Comment (alessio) [inline x/government/client/cli/query.go]:
```suggestion
		return fmt.Errorf("could not get government address: %s", err)
```

Let's avoid multiline output whenever possible

File: x/government/client/cli/query.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+func getCmdGetGovernmentAddrFunc(cmd *cobra.Command, args []string, cdc *codec.Codec) error {
+	cliCtx := context.NewCLIContext().WithCodec(cdc)
+
+	route := fmt.Sprintf("custom/%s/%s", types.QuerierRoute, types.QueryGovernmentAddress)
+	res, _, err := cliCtx.QueryWithData(route, nil)
+	if err != nil {
+		return fmt.Errorf("could not get government address: %s", err)
+	}
+
+	fmt.Println(string(res))
```
Comment (alessio) [inline x/government/client/cli/query.go]:
```suggestion
	cmd.Println(string(res))
```

File: x/government/module.go [hunk 4]
```diff
@@ -3,6 +3,10 @@ package government
 import (
 	"encoding/json"
 
+	"github.com/commercionetwork/commercionetwork/x/government/client/rest"
+
```
Comment (alessio) [inline x/government/module.go]:
```suggestion
```

Comment (gsora) [reply x/government/module.go]: Uh? Should I remove that import? Without that, I cannot register rest routes though.

Comment (alessio) [reply x/government/module.go]: just the empty line

File: x/government/client/cli/query.go (hunk 3 above)
Comment (alessio) [reply x/government/client/cli/query.go]: This is not solved yet

**Pull request merged**